### DISCUSSION TITLE
Applicazione IOS

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#151] [Idee e proposte]
Sarebbe possibile creare un applicazione per IOS come per ANDROID?

**Comment by trinko** [comment]
No, non c'è nessun progetto previsto per IOS.
Mi spiace

**Comment by zerai** [comment]
Eventualmente si potrebbe pensare di sviluppare e documentare (openApi) delle api per il mobile,
 questo aprirebbe la porta a progetti/versioni mobile mantenute da altri, una mi pare sia già attiva 
con una soluzione scraping+app.

**Reply by manuelbarberi** [reply]
Quale soluzione è attiva già ?

**Reply by MatteoGheza** [reply]
Infatti, sarebbe un'ottima cosa realizzare della documentazione OpenAPI.
Questo permetterebbe di realizzare app supportate dalla community, per esempio scritte in Flutter, per supportare sia Android che IOS.

**Reply by zerai** [reply]
@manuelbarberi  https://giua-app.github.io/

**Comment by zerai** [comment]
Magari si potrebbe iniziare con l'aprire una nuova discussione e identificare/pianificare i punti/step necessari alla realizzazione,
e le persone disponibili a collaborare con @trinko che mi pare già oberato con altri aspetti del software.
